insights/parsers/kdump.py

```diff
@@ -237,10 +237,20 @@ def fs_and_partation(self):
         """
         Return (fs_type, partation) as a tuple if set, else None.
         https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/kernel_administration_guide/kernel_crash_dump_guide#sect-supported-kdump-targets
+        If more than one target is configured, kudmp service fails to start.
         """
-        for t in self.SUPPORTED_FS_TYPES:
-            if t in self.data:
-                return (t, self.data[t])
+        targets = []
```

Comment:
I think the original `fs_and_partation` is good enough. in the contrary better do the conflict check in `parse_content`